### PR TITLE
Adds mhv-inherited-proofing application

### PR DESCRIPTION
**Pull request by AGarcia-Clarity**
## Description
This PR adds the MHV Inherited Proofing application to the content build.

vets-website PR for MHV Inherited Proofing application: https://github.com/department-of-veterans-affairs/vets-website/pull/20746

## Testing done
n/a

## Screenshots
n/a

## Acceptance criteria
- [x] An entry for the MHV Inherited Proofing is created and does not allow release to production

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/registry.json [hunk 1]
```diff
@@ -1214,5 +1214,15 @@
...
+    "entryName": "mhv-inherited-proofing",
+    "rootUrl": "/transfer-account",
+    "template": {
+      "vagovprod": false,
+      "title": "Transfer Account",
+      "layout": "page-react.html"
+    }
   }
-]
+]
```
Comment (theodur) [inline src/applications/registry.json:1228]:
```suggestion
]

```
Looks like the newline was removed

**Review by theodur — APPROVED** [review]
CI should pass once the `vets-website` PR is merged and the commit in `master` finishes

File: src/applications/registry.json (hunk 1 above)
Comment (AGarcia-Clarity) [reply src/applications/registry.json:1228]: 👀

**Comment by AGarcia-Clarity** [comment]
@theodur I merged the vets-website earlier today, just trying to verify this is working as intended (the CI for dev has been running for over an hour now 👀 ).  If you have time tomorrow.

**Comment by theodur** [comment]
@AGarcia-Clarity We were having issues with dev/staging deploys at the time, so the assets for the app weren't in the S3 bucket. I see that they're in there now, so you should be good to go. The registry was updated earlier today, so I merged `master` and resolved the conflicts for you

**Pull request merged**